kernel/src/fs/procfs/pid/status.rs

```diff
@@ -101,15 +101,17 @@ impl FileOps for StatusFileOps {
 
         {
             let vmar = process.lock_root_vmar();
-            let anon = vmar.unwrap().get_rss_counter(RssType::RSS_ANONPAGES) * (PAGE_SIZE / 1024);
-            let file = vmar.unwrap().get_rss_counter(RssType::RSS_FILEPAGES) * (PAGE_SIZE / 1024);
-            let rss = anon + file;
-            writeln!(
-                status_output,
-                "VmRSS:\t{} kB\nRssAnon:\t{} kB\nRssFile:\t{} kB",
-                rss, anon, file
-            )
-            .unwrap();
+            if let Some(vmar_ref) = vmar.as_ref() {
+                let anon = vmar_ref.get_rss_counter(RssType::RSS_ANONPAGES) * (PAGE_SIZE / 1024);
+                let file = vmar_ref.get_rss_counter(RssType::RSS_FILEPAGES) * (PAGE_SIZE / 1024);
+                let rss = anon + file;
+                writeln!(
+                    status_output,
+                    "VmRSS:\t{} kB\nRssAnon:\t{} kB\nRssFile:\t{} kB",
```

Comment:
I've confirmed this is the right Linux behavior. I think this should be part of the PR description.

```c
// z.c
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>

int main(void)
{
	pid_t pid = fork();

	if (pid == 0) {
		exit(EXIT_SUCCESS);
	}

	printf("%d\n", pid);
	sleep(1);

	return 0;
}
```

```shell
> gcc z.c -o z
> ./z
309078
^Z
[1]  + 309077 suspended  ./z
> cat /proc/309078/status | grep 'Vm'
> cat /proc/self/status | grep 'Vm'
VmPeak:	   5828 kB
VmSize:	   5828 kB
VmLck:	      0 kB
VmPin:	      0 kB
VmHWM:	   3528 kB
VmRSS:	   3528 kB
VmData:	    468 kB
VmStk:	    132 kB
VmExe:	     20 kB
VmLib:	   1644 kB
VmPTE:	     40 kB
VmSwap:	      0 kB
```

Reply:
In Linux such process is in `zombie` state, and does not have the  `VmRSS` value.

How to make it part of the PR description? Shall I add this in the code comment, or git message, or GitHub description?